### PR TITLE
build: migrate mandelbrot bundling from gulp to webpack

### LINKED ISSUE #585: Upgrade/replace Chokidar and FSEvents
Whilst installing Fractal on a new project, I noticed the following notice:

image

I'm _pretty sure_ fsevents is a security issue due to something about insecure binaries on AWS. I'm not 100% sure of the ins and outs - maybe mentioning @paulmillr will work in a repo he's not part of :crossed_fingers:.

Node 14 it going to become the new LTS shortly, so it's imperative that Fractal works on it.

I'll take this one on, as it feels like a good opportunity to work out some of the kinks and dodgy old code at the same time.

### PR DESCRIPTION
**Pull request by mihkeleidast**
Resolves #585.

### CODE REVIEW

**Review by LeBenLeBen — CHANGES_REQUESTED** [review]

File: packages/mandelbrot/assets/scss/components/_frame.scss [hunk 1]
```diff
@@ -80,7 +80,7 @@
     overflow: auto;
 
     &.is-loading {
-        background-image: url("../img/loader.svg");
+        background-image: url("../../img/loader.svg");
```
[the remark this answers is not in the document]
Comment (mihkeleidast) [reply packages/mandelbrot/assets/scss/components/_frame.scss:83]: shouldn't affect, the build output should remain the same - the change is only for webpack so it'll find the assets.

File: packages/mandelbrot/assets/js/mandelbrot.js [hunk 2]
```diff
@@ -1,10 +1,7 @@
 'use strict';
 
-import 'core-js/features/array/find';
```
[the remark this answers is not in the document]
Comment (mihkeleidast) [reply packages/mandelbrot/assets/js/mandelbrot.js:3]: I think I tested on IE11 and it was fine?

[the remark this answers is not in the document]
Comment (LeBenLeBen) [reply packages/mandelbrot/assets/js/mandelbrot.js:3]: `Array.find` is only used in specific places such as the search, the error shows up only once you interact with such features.

Screenshot_2020-10-24 Dashboard

File: packages/mandelbrot/assets/scss/components/_frame.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (LeBenLeBen) [reply packages/mandelbrot/assets/scss/components/_frame.scss:83]: Yeah I actually check a bunch of projects I have that extend Mandelbrot by importing Sass partials directly but those have the assets copied over already so it shouldn’t be a problem.

File: packages/mandelbrot/assets/js/components/preview.js [hunk 3]
```diff
@@ -40,7 +39,7 @@ class Preview {
         });
 
         this._resizer.resizable({
-            handleSelector: this._handle,
+            handleSelector: '> [data-role="resize-handle"]',
```
[the remark this answers is not in the document]
Comment (mihkeleidast) [reply packages/mandelbrot/assets/js/components/preview.js:42]: Good find, thanks.

File: packages/mandelbrot/webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    ],
+    module: {
+        rules: [
+            {
+                test: /\.m?js$/,
+                exclude: /(node_modules|bower_components)/,
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        presets: ['@babel/preset-env'],
```
[the remark this answers is not in the document]
Comment (mihkeleidast) [reply packages/mandelbrot/webpack.config.js:52]: Yup, using ES6 import/export resolves this.

File: packages/mandelbrot/assets/js/mandelbrot.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (mihkeleidast) [reply packages/mandelbrot/assets/js/mandelbrot.js:3]: Good find, thought I tested this before removing but apparently missed it. Either way, letting Babel automatically polyfill things like this makes sense, so I did that.

**Review by LeBenLeBen — APPROVED** [review]

**Pull request merged**